source/asdf/serialization.d

```diff
@@ -1522,6 +1522,16 @@ void serializeValue(S, V)(ref S serializer, auto ref V value)
         value.serialize(serializer);
     }
     else
+    static if(__traits(hasMember, V, "toString"))
+    {
+        serializer.putValue(value.toString);
+    }
+    else
+    static if(__traits(hasMember, V, "getRawValue"))
+    {
+        serializer.putValue(value.getRawValue);
+    }
+    else
     {
```

Comment:
So, we don't actually want to blindly call toString if it is available – because the last leaf of this cascade is one that looks at objects and converts it into a dict, which is a relevant use case that gets cut off if toString is present.

The issue here is that our units types have no public properties to deserialize. So here we have 2 choices IMO:

1. Explicitly add support for our units types – Price, Qty, etc.

2. If the struct contains no serializable members, then call `to!string` on it and take that as the value.
3.